webui/react/src/components/NotebookModal.tsx

```diff
@@ -311,7 +320,7 @@ const NotebookForm:React.FC<FormProps> = (
     <LabelledLine
       content = {
         <InputNumber
-          defaultValue={1}
+          defaultValue={(fields.slots && fields.slots > 0) ? fields.slots : 1}
```

Comment:
**suggestion:** I think this can simplify to:
```
defaultValue={fields.slots || 1}
```
Give it a try and see if it works